modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -1093,6 +1101,45 @@ private boolean _isActiveProfile(Project project, String profileName) {
 		return false;
 	}
 
+	private void _mapGlobalJSScriptElementAttributesToString(
+		ClientExtension clientExtension) {
+
+		Map<String, Object> typeSettings = clientExtension.typeSettings;
+
+		Map<String, Object> scriptElementAttributes =
+			(LinkedHashMap<String, Object>)typeSettings.get(
+				"scriptElementAttributes");
+
+		if ((scriptElementAttributes == null) ||
+			scriptElementAttributes.isEmpty()) {
+
+			typeSettings.remove("scriptElementAttributes");
+
+			return;
+		}
```

Comment:
We should probably follow whatever the existing pattern is. @drewbrokke what do we normally do for objects and arrays that we already have in other client extension configurations? Do we throw an error if they are empty or just leave them and handle the empty object or array?

Reply:
There's no one pattern we can follow. `key:` is valid Yaml, but it just means the `key` has a `null` value. In some cases it may be meaningful so it is valid for those cases. It's up to us what it means in this case. 

It seemed to me that a key with no value would be useless, so my preference would be to fail and explain that if you _are_ going to include this key, you should have some values in it.